scripts/local_proxies.sh

```diff
@@ -5,10 +5,10 @@ SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 
 echo "adding proxies"
 
-for node in $( seq 0 3 ); do
-  NodeAddr="localhost:$(( 2000 + node * 2))"
-  ProxyAddr="http://localhost:$(( 2001 + node * 2))"
-  echo -n "Adding proxy for node $(( node + 1)): "
+for node in $(seq 0 3); do
+  NodeAddr="grpc://localhost:$((2000 + node * 2))"
```

Comment:
From our test yesterday, it seems that the node address is expected without the protocol (ie. grpc).

Reply:
Which test? With the changes in dela#fix-notls, the standard protocol is `grpcs`. If you want something else, it needs to be specified.

Reply:
But when we used this script together yesterday, it generated a 404 on the backend.